sql/core/src/test/resources/sql-tests/results/udf/udf-udaf.sql.out

```diff
@@ -51,7 +52,7 @@ SELECT default.udaf1(udf(int_col1)) as udaf1, udf(default.udaf1(udf(int_col1)))
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-Can not load class 'test.non.existent.udaf' when registering the function 'default.udaf1', please make sure it is on the classpath; line 1 pos 94
```

Comment:
Here it was `'default.udaf1'` previously. But the new message is `'udaf1'`: https://github.com/apache/spark/pull/29502/files#diff-2d3d6b47e6044b4d44590c5a73b7cd8bR46

Do you know why?

Reply:
For permanent, the old code path is:
```
create function // no class check
query on function -> lookup function -> fill database name if permanent -> register function -> check class
```
So the previously msg always contains database name.

Now the code path is:
```
create function -> check class
```

Reply:
then shall we fill in the default database when creating permanent functions, to make the error message better?

Reply:
Permanent function always need a database, it's ok to fill database name when creating.